src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,59 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private int supplySum = 0;
+    private int buySum = 0;
```

Comment:
make them local

Reply:
explain why to make these variables local since I use them in several methods